### ISSUE TITLE
Lang files don't fully work

### ISSUE THREAD

**Issue by MrNinja93** [TehNut-Mods/HWYLA#23]
Just an FYI was just testing out Spanish lang file and found the translations aren't taking effect in a few places one being the "modules" and subsequent screens Just in-case it was my derping I checked the original included French and Russian languages and they both have the same issue

**French**
![2016-11-11_06 54 49](https://cloud.githubusercontent.com/assets/21979276/20206945/e49fa696-a7db-11e6-98ce-1e4c5156c3a5.png)

Edit: French isn't fully translated but some of that page is.

**Custom Spanish**
![2016-11-11_06 59 43](https://cloud.githubusercontent.com/assets/21979276/20207034/770d9074-a7dc-11e6-88cf-0281b5211127.png)

**Comment by TehNut** [comment]
Is this for the option titles? Like `Jukebox disk`?

**Comment by MrNinja93** [comment]
Yep the options. etc for the screenshots its the vanilla one, from the French file lines 117-123

option.vanilla.spawntype=Spawner type
option.vanilla.growthvalue=Niveau de croissance
option.vanilla.leverstate=Etat Levier
option.vanilla.repeater=Répeteur retard
option.vanilla.comparator=Comparateur mode
option.vanilla.redstone=Redstone puissance
option.vanilla.jukebox=Jukebox disque

Edit: Also the back button in the modules "home screen" isn't changing from English (every other "back" button changes though I think)

**Comment by TehNut** [comment]
Oh boy, the GUI widget system... This'll be a nice fun jump down the rabbit hole...

**Comment by MrNinja93** [comment]
> Oh boy, the GUI widget system... This'll be a nice fun jump down the rabbit hole...

Fun isn't what comes to mind when I looked at those files.... Good luck!

**Comment by MrNinja93** [comment]
To save opening another issue when it's somewhat related.

In the lang files if something can't be translated do you want  it be included in the lang file in English or that line be deleted (Some current files are including some are deleting, "Metadata" being one but also "%s" "Vanilla" "Waila" etc) Better to check than make an assumption and be wrong.

**Comment by TehNut** [comment]
Remove the line. Lang files are loaded in "layers" with `en_US` at the bottom.

**Comment by MrNinja93** [comment]
So I went to test an incoming PR and I found some extra info for this issue. The lang files take effect in the "modules" screens only after a restart of the game after you've changed language, everything else is an instant sync (Though the one "back" button never changes- See screenshot)

![2016-11-11_23 56 24](https://cloud.githubusercontent.com/assets/21979276/20233727/89b67bbc-a86a-11e6-8eae-3b58f6f12e68.png)
)

**Comment by TehNut** [comment]
This should be fixed when I finish #22
